### PR TITLE
ADEN-10771 add medianet prebid

### PR DESCRIPTION
**Pull request by jakubbanas**

### CODE REVIEW

File: src/ad-bidders/prebid/adapters-registry.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ import {
 	Wikia,
 	WikiaVideo,
 } from './adapters';
+import { Medianet } from './adapters/medianet';
```
Comment (dmnsobczak) [inline src/ad-bidders/prebid/adapters-registry.ts:24]: Please group it into imports from `./adapters` above

File: src/ad-bidders/prebid/adapters-registry.ts [hunk 2]
```diff
@@ -48,6 +49,7 @@ class AdaptersRegistry {
 		Verizon,
 		Wikia,
 		WikiaVideo,
+		Medianet,
```
Comment (dmnsobczak) [inline src/ad-bidders/prebid/adapters-registry.ts:52]: Alphabetical order

File: src/platforms-shared/bidders-state-setup.ts [hunk 3]
```diff
@@ -45,6 +45,7 @@ export function setupBidders(context: Context, instantConfig: InstantConfigServi
 		context.set('bidders.prebid.telaria.enabled', instantConfig.get('icPrebidTelaria'));
 		context.set('bidders.prebid.triplelift.enabled', instantConfig.get('icPrebidTriplelift'));
 		context.set('bidders.prebid.verizon.enabled', instantConfig.get('icPrebidVerizon'));
+		context.set('bidders.prebid.medianet.enabled', instantConfig.get('icPrebidMedianet'));
```
Comment (dmnsobczak) [inline src/platforms-shared/bidders-state-setup.ts:48]: Alphabetical order

File: src/ad-bidders/prebid/prebid-models.ts [hunk 4]
```diff
@@ -37,4 +37,6 @@ export interface PrebidAdSlotConfig {
 	uid?: string | number;
 	unit?: string;
 	zoneId?: string;
+	cid?: string;
```
Comment (dmnsobczak) [inline src/ad-bidders/prebid/prebid-models.ts:40]: Alphabetical order

**Review by dmnsobczak — CHANGES_REQUESTED** [review]
See comments

File: src/ad-bidders/prebid/adapters-registry.ts (hunk 1 above)
Comment (dmnsobczak) [reply src/ad-bidders/prebid/adapters-registry.ts:24]: Oh, now I know why you did it this way. Add Medianet class here:
https://github.com/Wikia/ad-engine/blob/c5a28e26f14b6f7ce8f53b335098458a89bc9129/src/ad-bidders/prebid/adapters/index.ts#L18

Comment (jakubbanas) [reply src/ad-bidders/prebid/adapters-registry.ts:24]: Auto import 🤦‍♂️

**Review by dmnsobczak — APPROVED** [review]

**Pull request merged**